libs/ui/editor/src/lib/blocks/types.ts

```diff
@@ -977,6 +997,35 @@ export function blockForQueryBlock(block: FullBlockFragment | null): BlockValue
         }
       }
 
+    case 'TeaserListBlock':
+      return {
+        key,
+        type: BlockType.TeaserList,
+        value: {
+          blockStyle: block.blockStyle,
+          filter: block.filter,
+          skip: block.skip ?? 0,
+          take: block.take ?? 6,
+          teaserType: block.teaserType ?? TeaserType.Article,
+          teasers: block.teasers.map((teaser, index) => [
+            `${index}`,
+            {
+              ...teaser,
+              type:
```

Comment:
No clue why we rewrite the __typename to type, they essentially do the same thing. Had to do this as the typing for the input teasers uses this.